docs/mobile_sdk.md

```diff
@@ -25,17 +25,28 @@ target 'MyApp WatchKit Extension' do
 end
 ```
 
+From the command line, type "pod install". This will install the SDK for an iOS app with a watchOS 2 extension.  Use the workspace file Cocoapods creates for your app to open the project in Xcode.
+
+### Installing using Carthage
+
+Create a file in your app's project directory all it 'Cartfile'. Put the following lines in the Cartfile:
+```
+github "openwhisk//openwhisk-client-swift.git" ~> 0.1.0 # Or latest version
+```
+
+From the command line, type 'carthage update --platform ios'. Carthage downloads and builds the SDK, creates a directory called Carthage in your app's project directory, and puts an OpenWhisk.framework file inside Carthage/build/iOS. Add OpenWhisk.framework to the embedded frameworks in your Xcode project.
```

Comment:
Seems like things the user needs to type should be formatted with backticks like this: `carthage update --platform ios`